Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+JENKINS_NODE = env.JENKINS_NODE ?: 'master'
+
+// Text colours definition
+def GREEN( String msg ) { return "\u001B[32m${msg}\u001B[0m" }
+def YELLOW( String msg ) { return "\u001B[33m${msg}\u001B[0m" }
+def RED( String msg ) { return "\u001B[31m${msg}\u001B[0m" }
+
+def LOGS_PUSH( String str ) {
+  LOGS.push( str )
+}
+
+def LOGS_DUMP() {
+  echo LOGS.join("\n")
+}
+
+LOGS = []
+JENKINS_BUILD_USER = "Unknown"
+
+// The build pipeline
+pipeline {
+
+  // Specify the Jenkins to to run on
+  agent { node { label "${JENKINS_NODE}" } }
+
+  // Various build and build plugin's options
+  options {
+    ansiColor('xterm')
+  }
+
+  stages {
+
+    stage("Build") {
```

Comment:
as next PR, please update jenkins file to have ability to run several commands and some of them in parallel.

Reply:
@romani sure. I have already prepared the code change that runs multipke commands in Hyper.sh in parallel, but to test it I need this PR to be merged :)